### DISCUSSION TITLE
My certification application has been approved, but no email has been sent to me for more than five days to allow me to use my benefits

### DISCUSSION THREAD

**Discussion by Jerry-Xcq** [community/community#142376] [GitHub Education] [answered]
### Hi everyone,

thank you 
![capture_20241023175647043](https://github.com/user-attachments/assets/5e74c168-3700-4f37-a19d-7807ecd79907)

**Comment by davevad93** [comment]
Hi @Jerry-Xcq  

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Jerry-Xcq was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Jerry-Xcq** [reply]
thank you, But I have been waiting for a week, which doesn't match what you said

**Reply by davevad93** [reply]
Hey @Jerry-Xcq you're welcome, your application has been approved on October 16 and today we are at day 23, as you said it's been a week but it could take even up to 2 weeks, so be patient and wait a little longer! 👍

**Reply by JohnDelatto** [reply]
> Hi @Jerry-Xcq
> 
> When the benefits become available, the status label will turn purple and change to:
> 
> * `Congrats! Your request for @Jerry-Xcq was approved and your benefits are now available.`
> 
> The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.
> 
> For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268
> 
> And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :
> 
> * [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
> * [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Hey Dave

I have a pending request with support

But I can't reach out anymore can you please tell the team to restore my account status

**Comment by ojmarcelino** [comment]
First things first, congrats on being approved.
An easy and simple way to check is browsing through [Settings \ Profile](https://github.com/settings/profile), and check if "Display PRO Badge" is available.

Source: [Displaying badges on your profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile)

**Reply by Jerry-Xcq** [reply, marked as answer]
thank you

**Comment by tharun977** [comment]
If your certification application has been approved but you haven’t received an email to activate your benefits after five days, it's a good idea to check your spam or junk folder in case the email was filtered there. If it's still missing, reach out to the support team for the program through which you applied for certification. They should be able to assist in verifying your account and ensuring that you receive your benefits.

**Reply by Jerry-Xcq** [reply]
May i ask how to access the support team?
